app/containers/IntentService.jsx

```diff
@@ -0,0 +1,102 @@
+/** @jsx h */
+import { h, Component } from 'preact'
+
+import Loading from '../components/Loading'
+
+export default class IntentService extends Component {
+  constructor (props, context) {
+    super(props, context)
+    this.store = context.store
+
+    const {window} = props
```

Comment:
I think the question is: will we ever want to create a service for a `window` that is **not** the current window?

As far as I remember, there is no use-case for this, so I think the `window`parameter can indeed be moved to cozy-client-js.